Podverse/ClipsTableViewController.swift

```diff
@@ -18,44 +18,153 @@ class ClipsTableViewController: PVViewController {
     @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
     @IBOutlet weak var statusMessage: UILabel!
     @IBOutlet weak var retryButton: UIButton!
+    
+    @IBOutlet weak var clipQueryActivityIndicator: UIActivityIndicatorView!
+    @IBOutlet weak var clipQueryMessage: UILabel!
+    @IBOutlet weak var clipQueryStatusView: UIView!
+    
+    @IBOutlet weak var filterType: UIButton!
+    @IBOutlet weak var sorting: UIButton!
+    
+    var clipQueryPage: Int = 0
+    var clipQueryIsLoading: Bool = false
+    var clipQueryEndOfResultsReached: Bool = false
+    var filterTypeSelected: ClipFilterType?
```

Comment:
I noticed this variable does not really need to be null ever... I would suggest you set it here to be .allPodcasts and then also add a didSet{} to it and set the title of the filterType button to it every time it changes. Lots of repeating code can be omitted this way coz you dont have to worry about setting the button title every single time this variable changes.

Reply:
Good idea. I haven't been thinking about how I can use get { } and didSet { } with variables to save code 🤔